examples/llm/src/models/layers/gpt_blocks.py

```diff
@@ -27,6 +29,21 @@ def forward(self, x):
         return self.mlp_down(self.mlp_act(self.mlp_up(x)))
 
 
+class FusedGPTMLP(nn.Module):
+
+    def __init__(self, cfg: DictConfig, device: Optional[str] = None):
+        super().__init__()
+        self.dense_gelu_dense = DenseResGeluDense(cfg.d_model,
```

Comment:
can you raise a nice error somewhere in the chain if this is going to fail on forward?

Reply:
It should fail on init and check_if_dense_gelu_dense_installed will raise the error.